### PR TITLE
Bug fix snet account deposit

### PR DESCRIPTION
**Pull request by rajeevbbqq**
Mint only if network is Local only

### CODE REVIEW

File: packages/snet_cli/snet_cli/commands/mpe_account.py [hunk 1]
```diff
@@ -41,5 +40,10 @@ def transfer_in_mpe(self):
         self.transact_contract_command("MultiPartyEscrow", "transfer", [self.args.receiver, self.args.amount])
 
     def mint_token(self):
-        address = self.args.address
-        self.transact_contract_command("SingularityNetToken", "mint", [address, int(self.args.amount)])
+        rez = self.config.session_to_dict()
```
Comment (anandrgitnirman) [inline packages/snet_cli/snet_cli/commands/mpe_account.py:43]: we nned to remove this definition here , not expose it as a command in snet-cli

File: packages/snet_cli/snet_cli/commands/mpe_account.py [hunk 2]
```diff
@@ -41,5 +40,10 @@ def transfer_in_mpe(self):
...
+        network = rez['session']['network']
+
+        if network == 'local':
+            self.transact_contract_command("SingularityNetToken", "mint", [self.args.address, int(self.args.amount)])
```
Comment (anandrgitnirman) [inline packages/snet_cli/snet_cli/commands/mpe_account.py:47]: can we add this in the test case itself and remove referece to mint_token def in mpe_account.py

File: packages/snet_cli/snet_cli/commands/mpe_account.py (hunk 1 above)
Comment (rajeevbbqq) [reply packages/snet_cli/snet_cli/commands/mpe_account.py]: Removed the cli command

File: packages/snet_cli/snet_cli/commands/mpe_account.py (hunk 2 above)
Comment (rajeevbbqq) [reply packages/snet_cli/snet_cli/commands/mpe_account.py]: separated as test case

File: packages/snet_cli/test/functional_tests/mint/mint.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
+from packages.snet_cli.snet.snet_cli.utils.utils import get_web3, get_contract_object
+from web3.gas_strategies.time_based import medium_gas_price_strategy
+
+TRANSACTION_TIMEOUT = 500
+DEFAULT_GAS = 300000
+HTTP_PROVIDER = "http://localhost:8545"
+
+wallet_address_1 = "0x592E3C0f3B038A0D673F19a18a773F993d4b2610"
```
Comment (raamb) [inline packages/snet_cli/test/functional_tests/mint/mint.py:8]: Lets remove private keys from the code.

Comment (anandrgitnirman) [reply packages/snet_cli/test/functional_tests/mint/mint.py:8]: @rajeevbbqq , you can create a random pvt key and public key on the  fly , tools avb to do so

Comment (rajeevbbqq) [reply packages/snet_cli/test/functional_tests/mint/mint.py:8]: @raamb & @anandrgitnirman hardcoded keys are also referred to in other test cases, and other test cases will fail if we create a key on the fly.

Comment (raamb) [reply packages/snet_cli/test/functional_tests/mint/mint.py:8]: Ok sure. Lets go ahead with this for now

**Review by raamb — APPROVED** [review]

**Pull request merged**